### PR TITLE
Allow font-face to not define its path (for defining system font variations)

### PR DESCRIPTION
**Pull request by notandrew**

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/9d423ceade746ba26013f6b42cebe1e519011b17'>9d423ceade746ba26013f6b42cebe1e519011b17</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by ling1726** [comment]
Please add a PR description and changelog entry

File: packages/fluentui/react-northstar/src/themes/teams/components/Menu/menuStyles.ts [hunk 1]
```diff
@@ -1,49 +1,49 @@
...
-          border: `${v.borderWidth} solid ${primary ? v.primaryBorderColor : v.borderColor || colors.border}`,
-          borderRadius: siteVariables.borderRadiusMedium,
-        }),
-      ...(underlined && {
-        borderBottom: `${v.underlinedBottomBorderWidth} solid ${v.underlinedBorderColor}`,
-      }),
-    };
-  },
-};
+import { pxToRem } from '../../../../utils';
```
Comment (layershifter) [inline packages/fluentui/react-northstar/src/themes/teams/components/Menu/menuStyles.ts:1]: This looks strange, can you please revert this change?

Comment (notandrew) [reply packages/fluentui/react-northstar/src/themes/teams/components/Menu/menuStyles.ts:1]: Can't find a way to get rid of it. Maybe there's odd spaces in there. Just doing the PR over instead?
